api-documentation.raml

```diff
@@ -21,6 +21,23 @@ securitySchemes:
     settings:
       roles: []
 types:
+  Client:
+    type: object
+    discriminator: client_id
+    properties: 
+      client_id: string
+      name: string
+      surname: string
+      email: string
+      telephone: string
+      cpf: string
+    example:
+      client_id: "0c7ea244-24e2-478f-9278-2e051a7ba817"
+      name: "Felicity"
+      surname: "Jones"
+      email: "felicity@gmail.com"
+      telephone: "11984345678"
+      cpf: "34598712387"
   Loan:
```

Comment:
As is specified in the requirements, we should keep track of which the user a given loan belongs to. Maybe create an action in the UserViewset for it.

Reply:
The loan has a property that is a foreign key to the client_id. Are you talking about something other than that?